api/ticket.rst

```diff
@@ -1,17 +1,18 @@
 Ticket endpoints
 ================
 
-Zammad comes with many ticket related endpoints. 
-For better overview the grouped those up:
+Zammad comes with many ticket related endpoints.
+For better overview, they are splitted in different sections:
```

Comment:
[Split is an irregular verb](https://dict.leo.org/englisch-deutsch/split), so it should probably be `they have been split`, rather than `splitted`.